plugins/generator-1.15.2/forge-1.15.2/templates/particle.java.ftl

```diff
@@ -55,6 +55,8 @@ public class ${name}Particle extends ${JavaModName}Elements.ModElement{
 	@OnlyIn(Dist.CLIENT) private static class CustomParticle extends SpriteTexturedParticle {
 
 		private final IAnimatedSprite spriteSet;
+		private float angularVelocity;

```

Comment:
You should wrap this and constructor part in if too to omit this code when not in use (when both vel. and acc. equal 0). Applies to both versions. Does not change anything, just keeps the code a tad cleaner.